Confus/Common/TeamIdentifier.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+namespace ConfusShared
+{
+    /// <summary> The Team's Identifier. There are two teams, blue and red. A player has a team, flag has a team, UI has a team, etc. </summary>
+    enum class ETeamIdentifier
+    {
+        /// <summary> This object doesn't have a team identifier. </summary>
+        None = 0,
+        /// <summary> This object has a red team identifier. </summary>
+        TeamRed = 1 << 0,
+        /// <summary> This object has a blue team identifier. </summary>
+        TeamBlue = 1 << 1
+    };
+}
+
+namespace ConfusServer
+{
+	/// <summary> The Team's Identifier. There are two teams, blue and red. A player has a team, flag has a team, UI has a team, etc. </summary>
+	enum class ETeamIdentifier
+	{
+		/// <summary> This object doesn't have a team identifier. </summary>
+		None = 0,
+		/// <summary> This object has a red team identifier. </summary>
+		TeamRed = 1 << 0,
+		/// <summary> This object has a blue team identifier. </summary>
+		TeamBlue = 1 << 1
+	};
+}
+
+namespace Confus
+{
+    /// <summary> The Team's Identifier. There are two teams, blue and red. A player has a team, flag has a team, UI has a team, etc. </summary>
+    enum class ETeamIdentifier
+    {
+        /// <summary> This object doesn't have a team identifier. </summary>
+        None = 0,
+        /// <summary> This object has a red team identifier. </summary>
+        TeamRed = 1 << 0,
+        /// <summary> This object has a blue team identifier. </summary>
+        TeamBlue = 1 << 1
+    };
+}
```

Comment:
I understand the whole ConfusShared, but this is just duplicating it all over again. What's wrong with changing the namespace used in the code to 'ConfusShared'?